platform/x11/detect.py

```diff
@@ -111,22 +112,31 @@ def configure(env):
     #	env['OBJSUFFIX'] = ".nt"+env['OBJSUFFIX']
     #	env['LIBSUFFIX'] = ".nt"+env['LIBSUFFIX']
 
-    if (env["target"] == "release"):
+    if (env["use_lto"] == "yes"):
+        env.Append(CCFLAGS=['-flto'])
+        env.Append(LINKFLAGS=['-flto'])
+
+
+    env.Append(CCFLAGS=['-pipe'])
+    env.Append(LINKFLAGS=['-pipe'])
 
+    if (env["bits"] == "32"):
```

Comment:
I realized that we should do arch-dependent flag in a more careful manner. Just testing the bits would cause problems on 32-bit ARM machines such as Raspberry Pi, so I just left it out for a future PR